src/components/Notes/SearchBar/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { ReactComponent as SearchIcon } from '@assets/Search.svg';
+
+import { debounce } from 'utils/functions/debounce';
+import { noteFilterActions } from '@redux/slices/NoteFilterSlice';
+import { noteApi } from 'services/NoteService';
+import { useAppDispatch, useAppSelector } from '@redux/hooks';
+import { INotes } from '..';
+import { usePrevious } from 'utils/hooks/usePrevious';
+import {
+  Container,
+  SearchIconContainer,
+  StyledInput,
+  StyledStack,
+} from './styles';
+
+interface SearchBarProps {
+  setNotesLocal: (arg: INotes[]) => void;
+}
+
+const SearchBar = React.memo(({ setNotesLocal }: SearchBarProps) => {
+  const dispatch = useAppDispatch();
+  const filterParams = useAppSelector((state) => state.noteFilterReducer);
+  const { data: notes, refetch: refetchNotes } = noteApi.useGetPatientNoteQuery(
+    { ...filterParams }
+  );
+
+  const { t } = useTranslation();
+
+  const inputRef = React.useRef<HTMLInputElement>(null);
+
+  const handleContainerClick = () => {
+    if (inputRef.current) {
+      inputRef.current.focus();
+    }
+  };
+
+  const [value, setValue] = React.useState<string>('');
```

Comment:
mess, there should be some order, for example: state, hooks, handlers, don't code like that

Reply:
fixed